src/Program.cs

```diff
@@ -55,6 +55,11 @@ private static async Task<int> Main(string[] args)
                 {
                     Argument = new Argument<string?>() { Arity = ArgumentArity.ExactlyOne }
                 },
+                new Option(new[] { "--include-generated" }, Resources.Inlcude_generated_code_files_in_formatting_operations)
+                {
+                    Argument = new Argument<bool>(),
+                    IsHidden = true
```

Comment:
I marked it as hidden because I don't expect this to be super common and I didn't want to add more to the help screen. I can likely be talked down from this stance.